docs/getting-started/integrating-codacy-with-your-git-workflow.md

```diff
@@ -0,0 +1,49 @@
+1.  [Blocking merging pull requests](#blocking-pull-requests)
+
+## 1. Configuring quality settings {: id="configuring-quality-settings"}
+
+[Configure the quality settings](../repositories-configure/adjusting-quality-settings.md) of your project to define the gates (minimum acceptable values) and goals for quality metrics such as issues, duplication, complexity, and code coverage.
+
+!!! info "Using code coverage as a quality gate"
+    To use code coverage as a quality gate:
+
+    -   Ensure that code coverage reports are being generated and uploaded to Codacy
+    -   Enable **Diff coverage is under** or **Coverage variation is under** when configuring the quality settings
+
+!!! tip
+    Depending on the initial quality of your repositories and on your current team practices, the default gates may be too strict. Consider initially limiting gates to new critical issues, new security issues, and considerable drops in code coverage.
+
+![Configuring quality settings](../repositories-configure/images/quality-settings.png)
+
+## 2. Activating the Git provider integration {: id="git-provider-integration"}
+
+Activate the integration with your Git provider ([GitHub](../repositories-configure/integrations/github-integration.md), [GitLab](../repositories-configure/integrations/gitlab-integration.md), [Bitbucket](../repositories-configure/integrations/bitbucket-integration.md)) and enable the option **Status checks** (GitHub) or **Pull request status** (GitLab and Bitbucket). This instructs Codacy to report to your Git provider whether the status checks on a pull request are passing or failing, based on your quality settings.
+
+![Enabling your Git provider integration](../repositories-configure/integrations/images/github-integration.png)
+
+## 3. Blocking merging pull requests {: id="blocking-pull-requests"}
+
+Set up your Git provider to enforce Codacy's feedback and block merging pull requests with failing status checks ([GitHub](https://docs.github.com/en/repositories/configuring-branches-and-merges-in-your-repository/defining-the-mergeability-of-pull-requests/managing-a-branch-protection-rule), [GitLab](https://docs.gitlab.com/ee/user/project/merge_requests/merge_when_pipeline_succeeds.html#only-allow-merge-requests-to-be-merged-if-the-pipeline-succeeds), [Bitbucket](https://support.atlassian.com/bitbucket-cloud/docs/suggest-or-require-checks-before-a-merge/)).
+
+## Next steps
+
+**Congratulations!**  🎉 You've successfully integrated and set up your first repository.
+
+Before moving on, it's important that you:
+
+-   Validate that Codacy is reporting the intended status on your pull requests
+-   Review your [tool and code pattern settings](../repositories-configure/configuring-code-patterns.md) and [quality gate settings](../repositories-configure/adjusting-quality-settings.md)
+
+This is important to eliminate any false positives that could inadvertently block the work of your team.
```

Comment:
What if we move that last box to its own section? Trying this one here

Reply:
Checking that Codacy is reporting the correct status on pull requests and double-checking the tool and code patterns is an important step to do **before** blocking the merge of pull requests. This is intended as a last validation before customers run the risk of blocking the work of developers and [the feedback was provided by Customer Success](https://codacy.slack.com/archives/CA5GWEXSN/p1643299700029900?thread_ts=1643205653.029600&cid=CA5GWEXSN). Please revert this change. :arrow_backward: